diff -u libindicator-12.10.2+14.10.20140922/configure.ac libindicator-12.10.2+14.10.20140922-new/configure.ac
--- libindicator-12.10.2+14.10.20140922/configure.ac	2014-09-22 15:40:07.000000000 +0200
+++ libindicator-12.10.2+14.10.20140922-new/configure.ac	2015-12-08 12:32:27.593450441 +0100
@@ -68,7 +68,7 @@
 AM_CONDITIONAL(USE_GTK3, [test x = x3])
 
 LT_LIB_M
-LIBINDICATOR_LIBS+="$LIBM"
+LIBINDICATOR_LIBS+=" $LIBM"
 
 ##############################
 # Custom Junk
